src/read.c

```diff
@@ -4121,7 +4111,7 @@ avifResult avifDecoderNthImage(avifDecoder * decoder, uint32_t frameIndex)
     if ((nearestKeyFrame > (decoder->imageIndex + 1)) || (requestedIndex <= decoder->imageIndex)) {
         // If we get here, a decoder flush is necessary
```

Comment:
`avifDecoderFlush()` does not longer exists. I suggest replacing the word "flush" by a more verbose explanation, such as

```
// We need to start from a fresh key frame so discard the unused decoder state and its previous frames. This will force the setup of a new decoder instance in avifDecoderNextImage().
```

Reply:
Done.